### ISSUE TITLE
Registration section UI

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#10]
## User story
1. As a user
2. I want / need to see a registration section
3. So that I can enter my registration data 

## Acceptance criteria
* Registration section appears after hitting the register button
* Registration section provides fields to enter the registration data - email, name, password, company, initial admin (maybe a checkbox "I want to register my company with this service")
* Registration section contains a button to start the registration function
* Registration section should show some message after successful or failed registration

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
